src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx

```diff
@@ -127,6 +127,49 @@ You can request to increase the limits based on your plan. Contact the [technica
+
+- Content delivered using an edge functions
+
+When an edge application delivers content created through an edge function on Azion, the request flow is processed fully through Azion servers and delivered directly to users.
+
+![Overview of a request/response flow using Edge Functions](/assets/docs/images/uploads/data-transferred-edge-functions)
+
+- Content offloaded through cache
+
+When content is found in Azion's cache, meaning there was a HIT in Edge Cache, it's delivered directly by the edge, without searching for the content on the origin. For clients using Tiered Cache, if there's a MISS in the Edge Cache Layer but a HIT in the Tiered Cache layer, the content is still delivered directly by the edge.
+
+Content hosted through Object Storage can also be stored in cache and be delivered directly by the edge.
+
+![Overview of a request/response flow of HIT in Edge Cache](/assets/docs/images/uploads/data-transferred-hit-cache)
+
+- Content feteched from an origin
+
+When content isn't found in Azion's cache, meaning there was a MISS in Edge Cache, Azion searches and fetches the content from the origin to deliver it to the user. For clients using Tiered Cache, there can be a MISS in both the Edge Cache and the Tiered Cache layer, and the content is searched on the origin before being delivered by the edge.
+
+For clients using Object Storage, Azion can also search and fetch the content from the origin if there's a MISS in Edge Cache. In this case, the origin is the object storage that was configured in the product.
+
+![Overview of a request/response flow of MISS in Edge Cache](/assets/docs/images/uploads/data-transferred-miss-cache-image-processor)
+
+- Content using Image Processor fetched from an origin
+
+Similar to the **Content offloaded through cache** and **Content feteched from an origin** flows, when the Image Processor module is enabled, images can be searched on Edge Cache and Tiered Cached, if enabled, and in case of a miss, the image is fetched from the origin and delivered by the edge, and cached for future requests.
+
+![Overview of a request/response flow of MISS in Edge Cache with Image Processor module](/assets/docs/images/uploads/data-transferred-miss-cache-image-processor.png)
+
+Furthermore, enabling other features can affect how traffic is processed. For example, if you enable Gzip compression for files, Azion will verify if the compressed version of the file is in cache for any request. If it is, it's delivered to the user. If it isn't, Azion compresses the file, delivers it to the user and stores it in cache for future requests.
+
+When Real-Time Purge is used to remove an object using the URL or cache key from either the Tiered Cache or Edge Cache layers, subsequent requests for purged content will result in a cache MISS and will be redirected to the origin. Once the updated object is cached at the edge, in a cache HIT case, the content is retrieved from cache instead of the origin.
```

Comment:
```suggestion
When **Real-Time Purge** is used to remove an object using the URL or cache key from either the **Tiered Cache** or **Edge Cache** layers, subsequent requests for purged content will result in a cache *MISS* and will be redirected to the origin. Once the updated object is cached at the edge, in a cache *HIT* case, the content is retrieved from cache instead of the origin.
```